gtests/net/mptcp/fastclose/receive_fastclose_with_rst_v4.pkt

```diff
@@ -15,6 +15,11 @@
 +0.0     <                                .  1:1(0)           ack 1     win 256                              <mpcapable v1 flags[flag_h] key[ckey=2, skey]>
 +0     accept(3, ..., ...) = 4
 
+// the 4th ack, making the mptcp socket fully established, will trigger the add_addr
++0.0     <                                .  1:1(0)           ack 1     win 256                                         <dss dack4=1 nocs>
++0.0     >                                .  1:1(0)           ack 1                <add_address addr[saddr1] hmac=auto>
++0.0     <                                .  1:1(0)           ack 1     win 256    <add_address addr[saddr1] addr_echo,  dss dack4=1 nocs>
```

Comment:
note that dropping the ',  dss dack4=1 nocs' part here causes a pktdrill internal error:

receive_fastclose_with_rst_v4.pkt:21: error handling packet: (null)